person.rb

```diff
@@ -13,6 +15,11 @@ def can_use_services?
     true if is_of_age? || @parent_permission
```

Comment:
- [ ] Here, you are calling `is_of_age?` however, your method is named as `of_age?` it is raising an error
![ruby-4](https://user-images.githubusercontent.com/55635977/136336862-f2066eb3-c0c7-40f8-8af4-22354c67ceed.png)